packages/devtools_app/lib/src/ui/flutter/service_extension_widgets.dart

```diff
@@ -142,11 +142,11 @@ List<Widget> getServiceExtensionWidgets() {
       extensions: [performanceOverlay, slowAnimations],
     ),
     ServiceExtensionButtonGroup(
-      minIncludeTextWidth: 1200,
+      minIncludeTextWidth: 1400,
```

Comment:
this change is required when experimental flag is on since new button is being added

Reply:
that is fine. All these constants are adhoc. As you add more ui they are expected to have to change.